src/pages/ResultsPage/ResultsPage.js

```diff
@@ -0,0 +1,181 @@
+import { Button, Flex, Stack, Text, Icon, Box } from '@chakra-ui/react';
+import { CustomButton, Section } from 'components';
+import arrowSvg from 'img/sprite.svg';
+import React, { useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { selectIsLoggedIn } from 'redux/user/slice/slice';
+
+const testScore = 60;
+const percentCorrectAnswers = 80;
+
+export const ResultsPage = () => {
+  const isLoggedIn = useSelector(selectIsLoggedIn);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (!isLoggedIn) {
```

Comment:
remove useEffect. It should be on home page or on button Finish, currently it is not allowing me to watch the page. It redirects me constantly on home page